### PR TITLE
Bug 1967842: Removed log messages from OCS cluster validation to prevent spamming of logs.

### PR DESCRIPTION
**Pull request by priyanka19-98**
Signed off by: Priyanka Jiandani <pjiandan@redhat.com>

### CODE REVIEW

**Review by nbalacha — CHANGES_REQUESTED** [review]
Please remove the logs from ocs_operator.go as well

File: internal/operators/ocs/ocs_operator.go [hunk 1]
```diff
@@ -77,20 +77,18 @@ func (o *operator) ValidateCluster(_ context.Context, cluster *common.Cluster) (
...
 	numOfHosts := len(cluster.Hosts)
 	if host.Inventory == "" {
-		o.log.Info("Empty Inventory of host with hostID ", host.ID)
 		return api.ValidationResult{Status: api.Pending, ValidationId: o.GetHostValidationID(), Reasons: []string{"Missing Inventory in some of the hosts"}}, nil
 	}
 	inventory, err := hostutil.UnmarshalInventory(host.Inventory)
 	if err != nil {
-		o.log.Errorf("Failed to get inventory from host with id %s", host.ID)
-		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetHostValidationID()}, err
+		message:=fmt.Sprintf("Failed to get inventory from host with id %s", host.ID)
```
Comment (nbalacha) [inline internal/operators/ocs/ocs_operator.go:84]: Nit: formatting. Please leave a space before and after ":="
Same comment holds elsewhere.

Comment (priyanka19-98) [reply internal/operators/ocs/ocs_operator.go]: done

**Review by nbalacha — APPROVED** [review]

**Review by gamli75 — APPROVED** [review]

**Comment by priyanka19-98** [comment]
@nbalacha  @gamli75  could you approve it again? I had to rebase this PR against previous OCS PR.

**Pull request merged**